lib/autoproj/daemon/package_repository.rb

```diff
@@ -39,6 +47,21 @@ def buildconf?
             def branch
                 vcs[:remote_branch] || vcs[:branch] || 'master'
             end
+
+            # @return [String]
+            def head_sha
+                pkg = autobuild
+                pkg.importdir ||= local_dir # TODO: What if importdir != srcdir?
```

Comment:
`importdir` is a horrible hack that was introduced at a time where I didn't want to fight back. Basically `importdir` is what the importer uses, not `srcdir`.

Reply:
That's the point.. The installation-manifest stores srcdir.. Since we can't load the workspace (we can't assume it's not broken) how do I guarantee that I am creating the importer with the same path it actually used to checkout the package?

Reply:
Then add importdir to the installation manifest.

Reply:
Sounds good.. Could you please create an issue in autoproj and assign me? Thanks